src/errors.rs

```diff
@@ -362,6 +362,10 @@ error_chain! {
         BrokenPartialFile {
             description("partially downloaded file may have been damaged and was removed, please try again")
         }
+        Generic(v: String) {
+            description("generic error")
+            display("encountered error: {}", v)
+        }
```

Comment:
I think this change is a bogon.  I can't find use of this anywhere.

Reply:
Seems so. Removing it!